test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -23,13 +23,18 @@
 
 /**
  * @test
+ * @bug 8294610
```

Comment:
is this tag valid for a test fix -- afaik it is for a "product code fix" i.e. the test is for a fix in the product code identified by the bugid ?

Reply:
JEP 425 added a lot of new tests but missed in `@bug` tag in many cases. It should be `@bug 8284161` (not 8294610). We can add 8294610 to the list, which I think is what you are saying.

Reply:
Hello Mark, it hasn't always been clear to me when to update the `@bug`. The contribution guide isn't completely clear either. So I usually update the bug id if the test change is substantial (in the way it affects the test). In this case, I've now taken Alan's suggestion to include the  `8284161` (the one which introduced the test for feature testing) along with the current bug id `8294610`. I am willing to drop the `8294610` from the `@bug` if there's some agreement towards it.

Alan has marked the JBS issue as `noreg-self`, so that's been taken care of.

Reply:
I think adding the original 8284161 is sufficient, but remove 8294610 as the convention has been not to include a test fix bugid in the @bug statement ... this is a correction I've had to apply in the past

Reply:
I agree with Mark. The `@bug` tag serves to identify the test that needs to be run for a particular source code change. If the changes are all in the test itself, then there's no need for that information. The JBS issue will have a noreg-self keyword indicating that the test itself needs to be run, and that there's no need for any new non-regression test to test the change.

Reply:
Done. Updated the PR to remove 8294610 from the bug id